data/geany.glade

```diff
@@ -3498,6 +3516,7 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="tooltip_text" translatable="yes">Display height in rows for the autocompletion list</property>
+                                        <property name="invisible_char">●</property>
```

Comment:
Odd.  But don't add it to the patch though, we'll see what's what later, but that's noise here.